apps/portals/src/configurations/nf/synapseConfigs/tools.ts

```diff
@@ -13,7 +13,6 @@ import {
   usageRequirementsSql,
   vendorSql,
   catalogNumberSql,
-  mtaRequiredSql,
```

Comment:
Unrelated to this PR - `mtaRequiredSql` was removed in #287, so just remove it from the import too